lib/c-api/Cargo.toml

```diff
@@ -39,6 +39,12 @@ thiserror = "1"
 typetag = { version = "0.1", optional = true }
 paste = "1.0"
 
+[dependencies.pirita]
+path = "../../../pirita/crates/pirita"
```

Comment:
What will happen if the directory is not found, since the dependency is optional... will it fail?

Reply:
Yes. We fixed that by just copy-pasting the entire pirita codebase into this repository.

It does compile when the /pirita is in the correct path, but I didn't want to inline the entire pirita repo in this commit. So that we have two commits at least: one with the diff of all the create-exe / create-obj / run-pirita changes and one with the /pirita repo.

I am still not sure how to solve the issue of not being able to depend on private repositories in CI.

Reply:
I've fixed this with a `ssh://` URL - it works locally but we need to look at how to integrate it in the CI via Deploy Keys / SSH keys